develop/devguide/Connector/AdvancedDataMinerDataPersistencePersistingTables.md

```diff
@@ -38,10 +38,24 @@ For example, the following table parameter defines some columns to be saved. Col
 
 ## Volatile tables
 
-In case you do not want anything to be kept in the database, you can use the volatile option (options attribute of the ArrayOptions tag).
+In case you do not want anything to be kept in the database, you can use the volatile option (*options* attribute of the *ArrayOptions* tag).
```

Comment:
```suggestion
In case you do not want anything to be kept in the database, you can use the [volatile](xref:Protocol.Params.Param.ArrayOptions-options#volatile) option.
```